internify/src/components/organisms/AuthNavbar.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./styles/AuthNavbar.css";
+import MenuIcon from "@material-ui/icons/Menu";
+import Container from "@material-ui/core/Container";
+import Grid from '@material-ui/core/Grid';
+
+
+const Navbar = () => {
+  return (
+    <Container>
+      <Grid container xs = {12} sm = {12} md = {12} className = "navbar_container">
+
+        <Grid item xs = {6} sm = {6} md = {6} style={{height: 70, width: '100%'}} className="logo">
+          <Link className = "auth_navbar_logo" to="/">
+            Internify
+          </Link>
+
+
+        </Grid>
+        <Grid container xs = {6} sm = {6} md = {6} className="auth_navbar_right_items">
+          <Grid className="auth_navbar_right_item">
+          <Link className="auth_navbar_item" to="placeHolder">
```

Comment:
In the future refrain from giving class names like `left` or `right` items. It's more conventional to name containers based on what they contain rather than the direction they life. This is in part because UI designs can always change in the future.